common/util/webhook_utils.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/google/uuid"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func ValidateUUID(u string) error {
+	_, err := uuid.Parse(u)
+	return err
+}
+
+func ValidateURL(u string) error {
+	_, err := url.ParseRequestURI(u)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func CombineFieldErrors(fieldErrors field.ErrorList) error {
+
+	if len(fieldErrors) == 0 {
+		return nil
+	}
+
+	var errorStrings []string
+	for _, fe := range fieldErrors {
+		errorStrings = append(errorStrings, fe.Error())
+	}
+	return errors.New(strings.Join(errorStrings, "; "))
+}
+
+func IsWebhookEnabled(key string) bool {
+	val, ok := os.LookupEnv(key)
+
+	if !ok {
+		fmt.Printf("error reading %s env variable", key)
+		// safer to return "true" as default for "IsWebhookDisabled",
```

Comment:
I wanted to keep this method self-sufficient. This method should return the default value instead of the caller catching it and deciding what to do with it. This implementation keeps the client logic much simpler. I can log it's value in the caller code, but I would still prefer to keep the implementation details inside the util method. What do you think?

Reply:
Let me know if it makes sense, resolving the comment, feel free to reopen if you need to.